docs/nfl.html

```diff
@@ -161,7 +161,7 @@ <h1>Microsoft MakeCode Arcade</h1>
 
             <div id="simulator" role="application">
                 <iframe
-                    src="https://arcade.makecode.com/---run?id=_V45Uw47cb7f8&nofooter=1"
+                    src="https://arcade.makecode.com/---run?id=_YTpi08Lv4aXs&nofooter=1"
```

Comment:
why does the link needs to change?

Reply:
the game has a ref to the specific version, don't think we have a way to ref latest version of extensions that are separate repos